### DISCUSSION TITLE
Location Automatic Checkout vs. Manual Checkout

### DISCUSSION THREAD

**Discussion by eriegger** [lucaapp/security-concept#24] [General]
Hello,
just started to work with locations and Luca app.
I created a location, enabled Automatic Checkout, checked in via printed QR-code and tried to checkout manually (AUSCHECKEN (<--)).
Now I get an Info (Error) message my position is required to checkout.
(Info: Wir brauchen deine Position, um zu überprüfen, ob du dich bei befindest. Wähle in deinen Luca Standort Einstellungen "immer".)

Is it intended behavior to completely disallow manual checkout if location has automatic checkout enabled?

Thanks.

**Comment by sand2dorn** [comment]
Apparently, it is intended behavior AND will be fixed.

ck12chris = Produktmanager at nexenio
https://twitter.com/ck12chris/status/1373248613657231364

I'm assuming that's about iOS. That function is completely missing in Android as of now.
